miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/display/Displayer.kt

```diff
@@ -1,8 +1,14 @@
 package com.rakuten.tech.mobile.miniapp.display
 
-import com.rakuten.tech.mobile.miniapp.MiniAppView
+import android.content.Context
+import com.rakuten.tech.mobile.miniapp.MiniAppDisplay
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
 
-internal class Displayer {
+internal class Displayer(val context: Context) {
 
-    suspend fun getViewForMiniApp(basePath: String): MiniAppView = RealMiniAppView(basePath)
+    suspend fun createMiniAppDisplay(basePath: String): MiniAppDisplay =
+        withContext(Dispatchers.Main) {
+            RealMiniAppDisplay(context, basePath)
```

Comment:
You may want to have a context null validation here or exception handling.

Reply:
Thanks. Done.